test/constructed_images.jl

```diff
@@ -214,7 +214,7 @@ mutable struct TestType end
         orig_img = ImageMagick.load(joinpath(workdir, "2by2.png"))
         fn = joinpath(workdir, "2by2_fromstream.png")
         open(fn, "w") do f
-            ImageMagick.save(Stream(format"PNG", f), orig_img)
+            ImageMagick.save(Stream{format"PNG"}(f), orig_img)
```

Comment:
Interesting how much disruption this depwarn causes. Tests will fail unless FileIO >= 1.6, but I guess we can live with that, and it won't happen in CI or an update test environment.